gnome-image-installer/util/gis-unattended-config.c

```diff
@@ -173,3 +173,17 @@ gis_unattended_config_new (const gchar *file_path,
                          "file-path", file_path,
                          NULL);
 }
+
+/**
+ * gis_unattended_config_get_key_file:
+ *
+ * This is a transitional function which we should remove in favour of
+ * structured accessors.
```

Comment:
Nitpick: Maybe turn this into a `FIXME` for easier grepping in future.

Reply:
It is removed later in this patch series.